ext/pcre/php_pcre.c

```diff
@@ -1135,6 +1135,11 @@ static void php_do_pcre_match(INTERNAL_FUNCTION_PARAMETERS, int global) /* {{{ *
 		RETURN_FALSE;
 	}
 
+	if (start_offset == LONG_MIN) {
+		zend_argument_value_error(5, "must be greater than " ZEND_LONG_FMT, LONG_MIN);
+		RETURN_THROWS();
+	}
```

Comment:
```suggestion
	if (start_offset == LONG_MIN) {
		start_offset == LONG_MIN + 1;
	}
```

Reply:
not sure we should hide the issue

Reply:
I belive we should, user should be able to pass `ZEND_LONG_MIN`.